package.json

```diff
@@ -19,6 +18,8 @@
     "prettier-fix": "prettier \"./components\" --write --ignore-unknown",
     "sass": "sass",
     "storybook": "start-storybook --ci -s images,dist -p 6006",
+    "storybook:build": "npm run build && build-storybook -s images,dist -o .out",
```

Comment:
These are using `:` and the other commands are using `-`

Reply:
That's a good question. I think the "community standard" is to use `:` (see the husky commands) but our 4K build scripts documentation has dashes, so that's why I did those for the lint commands and stuff... It would be good to consolidate to one or the other, and I'm personally in favor of the semicolon, but don't want to switch the others until/unless we decide to update our build script documentation.

Reply:
I don't think this is an either/or scenario. In my mind, the colon is used to signify a common prefix, or a group of scripts that are related, and the dash is essentially a stand-in for a space. So for example:

```json
"storybook:build": "things",
"storybook:build-prod": "things",
"storybook:start": "things",
"storybook:watch": "things",
```

Another example:

```json
"husky:commit-msg": "things",
"husky:pre-commit": "things",
"husky:pre-push": "things"
```

... another more theoretical example:

```json
"emulsify:lint": "things",
"emulsify:lint-css": "things",
"emulsify:build": "things",
"emulsify:start": "things",
"emulsify:format-ts": "things",
"emulsify:format-styles": "things",
"emulsify:storybook:lint-accessibility": "things",
```

To my knowledge, this is more of a convention than a standard, but it is relatively common.